routes/brands.router.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const router = express.Router();
+
+const Brand = require("../models/brand.model")
+
+router.get("/", async (req, res) => {
+  const brands = await Brand.find();
+  res.json({
+    success: true,
+    brands: brands
+  })
+});
+
+router.post("/", async (req, res) => {
+  let brands = [];
+  brands = await Brand.insertMany(req.body.brands);
+  res.json({
+    success: true,
+    data: {
+      message: "Brands created",
+      brands: brands,
+      count: brands && brands.length
+    }
+  });
+});
```

Comment:
Hey,
Not sure if it is a best practice, but wouldn't chaining the route with the same path help us structure the code better.
Something like this

```
router.route("/")
.get((req,res) => {
   code
})
.post((req,res) => {
  code
})
```

Or is there a reason so as to not do it?

Reply:
Thanks for the review. Yes I agree that chaining will provide a better structure. I'll do the changes in all files.